### ISSUE TITLE
Issue about Number of Epochs

### ISSUE THREAD

**Issue by zqiao11** [SJTU-DMTai/OnlineTSF#2]
Hi @MogicianXD.  I have a question regarding the number of epochs used in this work. I couldn’t find this information in the paper. It seems that you use a large number of epochs for training, rather than a single epoch as in standard online learning setups. Could you clarify this?

**Comment by MogicianXD** [comment]
Hi, thanks for your interest! We only perform multi-epoch training in the offline setting to train our model. (It is the same for any other time series forecasters to learn parameters.) After the model has been deployed online, we use [one-step gradient descent](https://github.com/SJTU-DMTai/OnlineTSF/blob/530a74c7a242c6653f6703cbf8e5954954f2a74c/exp/exp_online.py#L172) throughout online learning, i.e., a single epoch as you mentioned.

**Comment by zqiao11** [comment]
Thank you for your reply :) Now it is clear to me. 

Just another quick question: For these LSF datasets, a dataset-level normalization is conducted in data preprocessing, which uses the mean and std computed on the warmup phase. Is it correct?

**Comment by MogicianXD** [comment]
Yes! It is usually necessary to do so. 
What's more, it would be better to perform [RevIN](https://openreview.net/forum?id=cGDAkQo1C0p) on each input sample. The official implementation of PatchTST has incorporated RevIN, and we also apply it to other models.

**Comment by zqiao11** [comment]
Thanks a lot. Everything is clear now. Issue closed.
